packages/repository-json-schema/src/build-schema.ts

```diff
@@ -348,6 +348,7 @@ export function modelToJsonSchema<T extends object>(
   if (meta.description) {
     result.description = meta.description;
   }
+  result.type = 'object';
```

Comment:
Isn't the `type` default to `object` if `properties` is present?

Reply:
I am not sure, but the test fails when the element is not present so the validation does not work as expected.